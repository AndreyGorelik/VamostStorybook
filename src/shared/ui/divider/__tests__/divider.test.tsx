import React from 'react';
import renderer from 'react-test-renderer';
import 'jest';

import Divider from '../divider.component';

describe('<Divider />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Divider />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
