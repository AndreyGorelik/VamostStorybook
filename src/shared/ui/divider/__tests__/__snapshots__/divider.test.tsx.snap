// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> renders correctly 1`] = `
<View
  style={
    {
      "alignSelf": "center",
      "backgroundColor": "rgba(0, 0, 0, 0.19)",
      "borderRadius": 5,
      "height": 1,
      "marginVertical": 3,
      "width": "100%",
    }
  }
/>
`;
